### DISCUSSION TITLE
Who is considered the first computer programmer?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#685] [Galaxy Trivia] [answered]
**Bonus:** What job title would you give this person, considering YOE? (*unleash your creativity* 🤭)

**Comment by DevElkami** [comment]
Augusta Ada King, Countess of Lovelace, wrote the first computer programme in the world.

Join Ada team's in her horse counting program. :laughing:

**Comment by peterbaccaro** [comment]
**Ada Lovelace** is considered the first computer programmer. Even though she wrote about a computer, the Analytical Engine, that was never built, she realized that the computer could follow a series of simple instructions, a program, to perform a complex calculation.
Given her pioneering work in computing and programming, a good job title for Ada might be **Technology Visionary** or **Computer Programming Pioneer**.

**Comment by SANKARAMDAS** [comment]
Ada Lovelace
In her now famous note "G", Lovelace also adds a step-by-step description for computation of Bernoulli numbers with Babbage's machine - basically an algorithm - which, in effect, make her the world's first computer programmer. Ms Siffert, what fascinates you about Ada Lovelace?

**Comment by thuongtruong1009** [comment, marked as answer]
Ada Lovelace is considered the first computer programmer. She lived in the 19th century and worked on Charles Babbage's early mechanical general-purpose computer, the Analytical Engine.

Given Ada Lovelace's pioneering work in computer programming, her job title could be something like "Victorian Era Computing Pioneer" or "19th Century Computing Visionary". Other possibilities might include "Mechanical Computing Innovator" or "Analytical Engine Programming Trailblazer".
